mkosi/qemu.py

```diff
@@ -1112,7 +1115,10 @@ def run_qemu(args: Args, config: Config) -> None:
     cmdline += ["-cpu", "max"]
 
     if config.qemu_gui:
-        cmdline += ["-vga", "virtio"]
+        if config.architecture.is_x86_variant():
+            cmdline += ["-vga", "virtio"]
```

Comment:
Let's always use this as the fallback, no clue if it works on s390 or power or not but we might as well try this instead of doing nothing at all